### DISCUSSION TITLE
i can`t get what is going on with directories in vs code

### DISCUSSION THREAD

**Discussion by moray5554** [Cyfrin/foundry-full-course-cu#656] [General]
I have dir in my VS code /FOUNDRY-F23/foundry-simple-storage-f23/lib/forge-std/src/Script.sol
Why code (7.32.48) actually works?
```
import {Script} from "forge-std/Script.sol";
```
I think he missing "src" dir between them (bcs i don`t see any Script.sol inside "forge-std" directory

**Comment by shikhar229169** [comment]
In Solidity, when you import a file, the compiler searches for the imported file in the specified paths, including relative paths and the Solidity standard library. When you import a file using a relative path, like import "forge-std/Script.sol";, the compiler will start looking for the imported file in the current directory and its subdirectories.
